inst/qml/ContingencyTables.qml

```diff
@@ -139,16 +147,16 @@ Form
 		{
 			title: qsTr("Residuals")
 			debug: true
-			CheckBox { name: "residualsUnstandardized";			label: qsTr("Unstandardized")		}
-			CheckBox { name: "residualsStandardized";			label: qsTr("Standardized")			}
-			CheckBox { name: "residualsAdjustedStandardized";	label: qsTr("Adjusted standardized")	}
+			CheckBox { name: "residualUnstandardized";			label: qsTr("Unstandardized")		}
+			CheckBox { name: "residualStandardized";			label: qsTr("Standardized")			}
+			CheckBox { name: "residualAdjustedStandardized";	label: qsTr("Adjusted standardized")	}
```

Comment:
I am confused, why do we now have to "Redisuals" sections? It seems one of them is redundant?